CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 
 - Update to Wazuh version 3.9.4_7.2.0
 
+- Added Filebeat module and adapted Elasticsearch IP[rshad](https://github.com/rshad) [PR#144] (https://github.com/wazuh/wazuh-puppet/pull/144)
+
+- Added Kitchen testing for Wazuh deployment with Puppet. [rshad](https://github.com/rshad) [PR#139](https://github.com/wazuh/wazuh-puppet/pull/139)
```

Comment:
The merge of the Kitchen testing could take a while, we don't know. Let's remove it and include it in the proper PR #139 

Thanks

Reply:
Hi Jose,

Please take into account that all the changes related to `Kitchen` testing were already merged into the branch `3.9.4_7.2.0` before I derive `3.9.4_7.2.0_install_filebeat` from it.

Kind regards,

Rshad